plotly_kaleido/src/lib.rs

```diff
@@ -163,7 +163,7 @@ impl Kaleido {
         }
 
         let output_lines = BufReader::new(process.stdout.unwrap()).lines();
-        for line in output_lines.flatten() {
+        for line in output_lines.map_while(Result::ok) {
```

Comment:
Doesn't look related to issue title but plausibly just adding in a stray cleanup?

Reply:
Yes,  you are correct. Updated the title and added a bit more context to the PR description.

Rust comes with [`clippy`](https://doc.rust-lang.org/clippy/) , a very nice linter. The latest clippy introduced two more lints which were fixed in the second commit of this PR.